fgpyo/fasta/reference_set_builder.py

```diff
@@ -0,0 +1,174 @@
+"""
+Classes for generating Fasta files and records for testing
+----------------------------------------------------------------
+"""
+# import hashlib
+import os
+import textwrap
+from tempfile import NamedTemporaryFile
+from typing import List
+from typing import Optional
+
+# pylint: disable=W0511
```

Comment:
```suggestion
```

Reply:
Removed in commit cb8ac832a05ba4a3ad304edb2de19c57d6f9b8f3